xsos

```diff
@@ -2919,15 +2920,17 @@ SYSCTL() {
   __Pa vm.dirty_background_bytes ""  '{if ($1>0) printf "\"%s\"%s  (%.1f MiB)", $1, H0, $1/1024/1024; else printf "\"0\"%s  (disabled -- check dirty_background_ratio)", H0}'
   __P vm.dirty_expire_centisecs
   __P vm.dirty_writeback_centisecs
+  __P vm.max_map_count ""
+  __P vm.min_free_kbytes ""
   __Pa vm.nr_hugepages "[$hpgsz-MiB pages] "   "-vHPGSZ=$hpgsz"  '{if ($1>0) printf "\"%s\"%s  (%.1f GiB total)", $1, H0, $1*HPGSZ/1024; else printf "\"%s\"%s", $1, H0}'
   __Pa vm.nr_overcommit_hugepages "[$hpgsz-MiB pages] "   "-vHPGSZ=$hpgsz"  '{if ($1>0) printf "\"%s\"%s  (%.1f GiB total)", $1, H0, $1*HPGSZ/1024; else printf "\"%s\"%s", $1, H0}'
   __Pa vm.overcommit_memory "[0-2] "   '{if ($1==0) printf "\"0\"%s  (heuristic overcommit)", H0; else if ($1==1) printf "\"1\"%s  (always overcommit, never check)", H0; else if ($1==2) print "\"2\"%s  (always check, never overcommit)", H0}'
   __P vm.overcommit_ratio
   __Pa vm.oom_kill_allocating_task "[bool] "  '{if ($1==0) printf "\"0\"%s  (scan tasklist)", H0; else printf "\"1\"%s  (kill OOM-triggering task)", H0}'
   __Pa vm.panic_on_oom "[0-2] "  '{if ($1==0) printf "\"0\"%s  (no panic)", H0; else if ($1==1) printf "\"1\"%s  (no panic if OOM-triggering task limited by mbind/cpuset)", H0; else if ($1==2) printf "\"2\"%s  (always panic)", H0}'
   __P vm.swappiness "[0-100] "
-  __P vm.min_free_kbytes ""
   __P vm.vfs_cache_pressure "[0-100] "
+  __P vm.zone_reclaim_mode "[0-4] "
```

Comment:
@rhn-jaeshin From my reading of the kernel-docs, it's not accurate to say `[0-4]`. Looks like it's a bitmask from `0` to `7`.

Reply:
EDIT: Ahhhhh digging further, it seems it really is a bitmask. So you can set it to 0, 1, 3, 5, or 7.